website/index.html

```diff
@@ -271,6 +273,7 @@
 
 <div class="section-separator">
   <img class="section-separator first-border">
+  <img src = "assets/images/section-sep-1-mountain-night.svg">
```

Comment:
add this src to the above line instead. This will generate two images, one which will always be visible regardless.